### PR TITLE
Time slider in 3d

### PR DESCRIPTION
**Pull request by oterral**
[test](https://mf-geoadmin3.int.bgdi.ch/fix_3887/index.html?lang=fr&topic=luftbilder&bgLayer=voidLayer&dev3d=true&debug&layers=ch.swisstopo.lubis-bildstreifen,ch.swisstopo.swissnames3d&catalogNodes=1179,1180,1184&layers_timestamp=2007,&lon=7.67481&lat=45.92214&elevation=90269&heading=359.987&pitch=-40.673&time=2007)

### CODE REVIEW

**Comment by davidoesch** [comment]
Testlink?

**Comment by oterral** [comment]
in a close future

**Comment by oterral** [comment]
[test](https://mf-geoadmin3.int.bgdi.ch/fix_3887/index.html?lang=fr&topic=luftbilder&bgLayer=voidLayer&dev3d=true&debug&layers=ch.swisstopo.lubis-bildstreifen,ch.swisstopo.swissnames3d&catalogNodes=1179,1180,1184&layers_timestamp=2007,&lon=7.67481&lat=45.92214&elevation=90269&heading=359.987&pitch=-40.673&time=2007)

**Comment by davidoesch** [comment]
https://mf-geoadmin3.int.bgdi.ch/fix_3887/index.html?lang=fr&topic=luftbilder&bgLayer=ch.swisstopo.pixelkarte-farbe&dev3d=true&debug&layers=ch.swisstopo.lubis-bildstreifen,ch.swisstopo.swissnames3d,ch.swisstopo.lubis-luftbilder-dritte-kantone,ch.swisstopo.lubis-luftbilder-dritte-firmen,ch.swisstopo.lubis-luftbilder_farbe,ch.swisstopo.lubis-luftbilder_schwarzweiss&catalogNodes=1179,1180,1184&layers_timestamp=2011,,,,,19571231&lon=7.86696&lat=46.06666&elevation=83363&heading=359.987&pitch=-40.673&time=1957&layers_visibility=false,true,true,true,true,true

no data shown

**Comment by oterral** [comment]
I have data.

**Comment by davidoesch** [comment]
ok me too now...waited like 2 minutes! perfomance?

**Comment by oterral** [comment]
So what do we do. I agree it's really slow. Only solutions I see is to pre-generate all the tiles at least on low zoom level.

**Comment by loicgasser** [comment]
Make wmts_url point to prod bucket and see if you notice any differences.

**Comment by oterral** [comment]
Once it's in the cloud front cache it's acceptable.

File: Makefile [hunk 1]
```diff
@@ -344,18 +344,16 @@ olcesium: .build-artefacts/ol-cesium
...
 	cd ol; \
 	git reset HEAD --hard; \
 	git fetch --all; \
-	git checkout $(OL_VERSION); \
 	git show; \
 	npm install --only prod; \
 	node tasks/build-ext.js; \
 	cd ../cesium; \
 	git remote | grep c2c || git remote add c2c git://github.com/camptocamp/cesium; \
+	git remote | grep oterral || git remote add oterral git://github.com/oterral/cesium; \
```
Comment (loicgasser) [inline Makefile]: Is that just a convenient way to test thing directly from your repo or do we use it?

Comment (oterral) [reply Makefile]: It will be removed when the PR will be ready

Comment (oterral) [reply Makefile]: yes it for testing branch on my fork

**Comment by oterral** [comment]
What do we do  @davidoesch ?

**Comment by oterral** [comment]
@davidoesch?

**Comment by davidoesch** [comment]
give it a go
It is faster now

**Pull request merged**